bcda/cclf/cclf.go

```diff
@@ -651,14 +654,11 @@ func validateCCLFFolderName(folderName string) error {
 
 func cleanUpCCLF(ctx context.Context, cclfMap map[string]map[int][]*cclfFileMetadata) error {
 	errCount := 0
-	close := metrics.NewChild(ctx, "cleanUpAllCCLF")
-	defer close()
-
 	for _, perfYearCCLFFileList := range cclfMap {
 		for _, cclfFileList := range perfYearCCLFFileList {
 			for _, cclf := range cclfFileList {
 				func() {
-					close := metrics.NewChild(ctx, "cleanUpIndividualCCLF")
+					close := metrics.NewChild(ctx, fmt.Sprintf("cleanUpCCLF%d", cclf.cclfNum))
```

Comment:
The CCLF num is static across ACOs.  I wonder if it makes sense to instead incorporate the ACO ID for the specific CCLF here?  Or, is that already being captured somewhere?
`cclf.acoID`

Reply:
I think for a metric name, it'd make sense to use more generic fields (like CCLF numbers).

What's not captured here, and we may want to explore, is adding custom tags to the metrics. I think if we add tags to segments/transactions, we'd be able to see what specific ACO is associated with a slower transaction, while still being able to use generic aggregation.

Reply:
Makes sense!